pyk/src/tests/test_kllvm.py

```diff
@@ -0,0 +1,9 @@
+from unittest import TestCase
+
+import kllvm
+
+
+class KLLVMTest(TestCase):
+    def test_basic(self):
+        sort_int = kllvm.ast.CompositeSort('SortInt')
+        self.assertEqual(str(sort_int), "SortInt{}")
```

Comment:
Did you confirm that this test is being run on CI correctly (shows up in the output)?

Reply:
From the CI log:
```
test_basic (tests.test_kllvm.KLLVMTest) ... ok
```